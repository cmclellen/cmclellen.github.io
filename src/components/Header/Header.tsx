import React, { FC } from 'react';
import { HeaderWrapper } from './Header.styled';

interface HeaderProps {}

const Header: FC<HeaderProps> = () => (
 <HeaderWrapper data-testid="Header">
    Header Component
 </HeaderWrapper>
);

export default Header;
