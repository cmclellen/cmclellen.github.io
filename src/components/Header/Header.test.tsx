import React from 'react';
import renderer from 'react-test-renderer'
import { describe, expect, it } from 'vitest'
import Header from './Header';

describe('<Header />', () => {
  it('it should mount', () => {
    const component = renderer.create(<Header />);

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  });
});