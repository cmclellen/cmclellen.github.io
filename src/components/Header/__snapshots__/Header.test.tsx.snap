// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Header /> > it should mount 1`] = `
<div
  className="sc-bdvuGq hka-DTH"
  data-testid="Header"
>
  Header Component
</div>
`;
